### PR TITLE
Draft web app proposal.

### PR DESCRIPTION
**Pull request by alexhsamuel**

### CODE REVIEW

File: assignment-webproposal.html [hunk 1]
```diff
@@ -38,15 +38,90 @@
...
+    <h2>Presentation</h2>
+    <p>
+      You will present your proposal to a group of your peers during Day 2 of this workshop.  You'll go over
+      your written proposal, and discuss the user story, design, and interactivity of the app.
+      Your peers will ask questions about the proposal, as well as give constructive suggestions for improving
+      the proposal, which you'll discuss as a group.
+    </p>
+    <p>
+      You'll also have the opportunity to listen to your peers' proposals, ask questions about them, and
+      provide feedback with your own ideads of how to improve their proposals.
```
Comment (bensussman) [inline assignment-webproposal.html]: typo: `ideads` => `ideas`

File: assignment-webproposal.html [hunk 2]
```diff
@@ -38,15 +38,90 @@
...
+      </ul>
+    </p>
+    <p>
+      Presentations will be evaluated based on the following.
+      <ul>
+        <li>Is your presentation well-prepared, clear, and thorough?</li>
+        <li>Do you ask good questions about your peers' proposals?</li>
+        <li>Do listen carefully to questions and do your best to discuss them?</li>
+        <li>Do you provide constructive suggestions about your peers' proposals?</li>
+        <li>Do you accept and adopt your peers' suggestions for your proposal?</li>
```
Comment (bensussman) [inline assignment-webproposal.html]: How will we measure this in the 20 minutes? Adopting suggestions seems like a longer term thing.

**Comment by bensussman** [comment]
Do you want to link directly to your example proposal (the Rockaways)?

File: assignment-webproposal.html (hunk 2 above)
Comment (alexhsamuel) [inline assignment-webproposal.html]: Yeah, "adopt" isn't really the right word.

File: assignment-webproposal.html (hunk 1 above)
Comment (alexhsamuel) [inline assignment-webproposal.html]: Fixed.

**Comment by alexhsamuel** [comment]
Added link to Rockaway Lunch example.

**Comment by bensussman** [comment]
👍  LGTM. Is this ready to merge, or are you still working on this?

**Comment by alexhsamuel** [comment]
Let's merge, and if others have comments, I'll make a new PR.

You merge, or I?

**Comment by bensussman** [comment]
FYI i use http://gitbot.co/#home for deploying to GH Pages (super useful tool built by @mottaquikarim )

**Pull request merged**